generator/source/faq/consensus.rst

```diff
@@ -265,6 +265,13 @@ In PoET, forks occur due to a network partition, the size of the network, the ti
 
 TPs don’t really affect forks, unless they have a severe impact on the validation duration of the block. However, unresolvable forks due to non-determinism, are likely a TP problem.
 
+How do I fix PoET using Intel SGX IAS API version 2, which is end of life?
+--------------------------------------------------------------------------
+For those who are trying to use PoET SGX with IAS, you need to move to the IAS API v3 interface. Basically just go through and change the IAS URLs from `/v2/` to `/v3/`
+The change needs to be made in files `ias_client/sawtooth_ias_client/ias_client.py`
+and `ias_client/tests/unit/test_ias_client.py` .
```

Comment:
Can you also include a link to the v3 API:
https://software.intel.com/sites/default/files/managed/7e/3b/ias-api-spec.pdf

Reply:
Yes.